src/dct/templates/Template.lua

```diff
@@ -147,6 +151,9 @@ local function checkbldgdata(keydata, tpl)
 		local sceneryobject = { id_ = tonumber(bldgdata.data.name), }
 		utils.mergetables(bldgdata.data,
 			vector.Vector2D(Object.getPoint(sceneryobject)):raw())
+		if tpl.tpldata == nil then
```

Comment:
Probably better to just do the check once at the top of the function instead of checking each look iteration:

```
 local function checkbldgdata(keydata, tpl)
+	if next(tpl[keydata.name]) ~= nil and tpl.tpldata == nil then
+		tpl.tpldata = {}
+	end
+
 	for _, bldg in ipairs(tpl[keydata.name]) do
```